convert2rhel/backup/__init__.py

```diff
@@ -86,18 +87,18 @@ def pop_all(self):
         """
         Restores all RestorableChanges known to the Controller and then returns them.
 
-        :returns: List of RestorableChange objects that were known to the Controller.
+        :returns processed_restorables: List of RestorableChange objects that were processed by the Controller.
```

Comment:
In this case you don't need the variable name here, you should use the type, I believe: 

```suggestion
        :returns list[RestorableChange]: List of RestorableChange objects that were processed by the Controller.
```